tools/sphere.c

```diff
@@ -69,6 +69,17 @@ int do_sloop( string arg ) {
                 ,FINDP( arg )->query_name() ), BOLD_PURPLE );
         return 1;
     }
+    /**
+     * If already slooping victim, instead remove both shadows
+     */
+    if ( sizeof( find_slooper_shadow( slooper ) ) && sizeof( find_sloopee_shadow( sloopee ) ) ) {
```

Comment:
I know I already left this comment somewhere but in case you lost it, this check isn't right.
If A sloops B and C sloops D, then when A does 'sloop D' it'll tell A they stop slooping D even though they were never slooping D. If this is a pain to fix I wouldn't worry about it, because when we use real snoop the driver keeps track of that for us, but I thought I should point it out.